src/org/montclairrobotics/sprocket/pipeline/Pipeline.java

```diff
@@ -0,0 +1,34 @@
+package org.montclairrobotics.sprocket.pipeline;
+
+import java.util.ArrayList;
+
+public class Pipeline <T>{
```

Comment:
I don't like that at all. Everything that implements Updatable should use Updater. That would be confusing and not clear, and it erodes the standards we set.